src/test/java/test/EmployeeTest.java

```diff
@@ -0,0 +1,129 @@
+package test;
+
+import com.google.gson.Gson;
+import dto.*;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import org.assertj.core.api.SoftAssertions;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import static io.restassured.RestAssured.given;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class EmployeeTest {
+
+    private RequestSpecification requestSpecification;
+
+    @BeforeClass
+    public void createRequestSpecification() {
+        requestSpecification = new RequestSpecBuilder()
+                .setBaseUri("http://dummy.restapiexample.com/api/v1")
+                .build();
+    }
+
+    @DataProvider(name = "existingEmployees")
+    public Object[][] existingEmployees() {
+        return new Object[][]{
+                {EmployeeData.builder()
+                        .id(1)
+                        .employeeName("Tiger Nixon")
+                        .employeeSalary(320800)
+                        .employeeAge(61)
+                        .profileImage("")
+                        .build()},
+                {EmployeeData.builder()
+                        .id(2)
+                        .employeeName("Garrett Winters")
+                        .employeeSalary(170750)
+                        .employeeAge(63)
+                        .profileImage("")
+                        .build()},
+                {EmployeeData.builder()
+                        .id(3)
+                        .employeeName("Ashton Cox")
+                        .employeeSalary(86000)
+                        .employeeAge(66)
+                        .profileImage("")
+                        .build()},
+        };
+    }
+
+    @Test(dataProvider = "existingEmployees")
+    public void checkGetEmployeesByIds(EmployeeData employeeData) {
+        Employee employee = getEmployee(employeeData.getId());
+
+        SoftAssertions softly = new SoftAssertions();
+        softly.assertThat(employee.getData().getEmployeeName()).isEqualTo(employeeData.getEmployeeName());
```

Comment:
actually you might want to use assertj's isEqualToIgnoringGivenFields instead of specifying all the fields